Navigation/Screens/Expenses.jsx

```diff
@@ -1,9 +1,52 @@
 import React from "react";
 import { Text } from "react-native";
+import { Button } from "react-native";
+import { View } from "react-native";
+import { useState } from "react";
+import { FlatList } from "react-native";
+import { StyleSheet } from "react-native";
 
-
-export default function Expenses(){
+export default function Expenses(props){
+    const [allExpenses,setAllExpenses] = useState([
+        {title:'dosa',recipient:'leo',amount:'100',category:'food',date:'1/21/2024'},
+        {title:'auto',recipient:'white',amount:'60',category:'travel',date:'1/20/2024'},
+        {title:'books',recipient:'taki',amount:'200',category:'utilities',date:'1/15/2024'}
+    ])
```

Comment:
you can create a dummyData constant file and store all these data that you need for testing there and you can import and consume it from there
like

```
const expensesData = [
        {title:'dosa',recipient:'leo',amount:'100',category:'food',date:'1/21/2024'},
        {title:'auto',recipient:'white',amount:'60',category:'travel',date:'1/20/2024'},
        {title:'books',recipient:'taki',amount:'200',category:'utilities',date:'1/15/2024'}
 ]
```

Reply:
made the changes , I will push today evening